inc/template-functions.php

```diff
@@ -440,3 +438,22 @@ function twenty_twenty_one_print_first_instance_of_block( $block_name, $content
 
 	return false;
 }
+
+/**
+ * Retrieve protected post password form content.
+ *
+ * @since 1.0.0
+ *
+ * @param int|WP_Post $post Optional. Post ID or WP_Post object. Default is global $post.
+ * @return string HTML content for password form for password protected post.
+ */
+function twenty_twenty_one_password_form( $post = 0 ) {
+	$post   = get_post( $post );
+	$label  = 'pwbox-' . ( empty( $post->ID ) ? wp_rand() : $post->ID );
+	$output = '<p>' . __( 'This content is password protected. To view it please enter your password below:', 'twentytwentyone' ) . '</p>
```

Comment:
Should probably be escaped

```suggestion
	$output = '<p>' . esc_html__( 'This content is password protected. To view it please enter your password below:', 'twentytwentyone' ) . '</p>
```